demo_scripts/moodle/README.md

```diff
@@ -0,0 +1,72 @@
+# Deploy a scalable Moodle stack on Azure
+
+A pre-requisite to going through this tutorial is `az login`. Please run `az login` from the command line and get back to this deployment tutorial. Many commands in this deployment experience require you to specifiy values. In order to get going as quickly as possible we've provided defaults for you.
+
+Here are the defaults that will be used:
+```
+env | grep MOO
+```
+# Creating a local workspace for Moodle ARM and other configuration files
+```
+mkdir -p $MOODLE_AZURE_WORKSPACE/$MOODLE_RG_NAME
+```
+# We will use SSH to access infrastructure post-deployment and the following command will create a Secure Shell (SSH) key-pair to assist for when you're ready to remotely connect to your Moodle deployment via SSH
```

Comment:
This feels like it should be a prerequisite in the `Azure` directory since it is likely that a) many users will already have an SSH key and b) any scenarios will want to validate an SSH key is present.

By putting this into `Azure/SSH_Create` you can do

```
# Prerequisites

To create a Moodle cluster you will need an SSH key. If you don't have one take a look at the [Create an SSH Key](/Azure/SSH_Create/README.md") document.
````

In that document you verify that the key exists and, if it doesn't you create one (i.e. you have the same commands you have here). When it returns you will have the key in the environment variable.

Reply:
Was blindly following instructions in the Moodle setup. Will fix.